### DISCUSSION TITLE
Wildfly Administration Console: user name password

### DISCUSSION THREAD

**Discussion by nmdasay** [dcm4che/dcm4chee-arc-light#4332] [General]
Hi.
can not login in to Wildfly Administration Console using user :root , password: changeit

**Comment by skarramooch** [comment]
add the two bottom wildfly lines to the docker-compose.yml file and restart:

![image](https://github.com/dcm4che/dcm4chee-arc-light/assets/157669331/457f3620-2ae6-454e-b112-12428c0f064a)

**Reply by vrindanayak** [reply]
> add the two bottom wildfly lines to the docker-compose.yml file and restart:
> 
> ![image](https://private-user-images.githubusercontent.com/157669331/299533024-457f3620-2ae6-454e-b112-12428c0f064a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2MDY1MjAsIm5iZiI6MTcyOTYwNjIyMCwicGF0aCI6Ii8xNTc2NjkzMzEvMjk5NTMzMDI0LTQ1N2YzNjIwLTJhZTYtNDU0ZS1iMTEyLTEyNDI4YzBmMDY0YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyMlQxNDEwMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZDRlOTJlNTJmNzVhMzNkOWM4NmQzZmIyYmE4NmRhZDM5YTg2NmQ1OTE1MTFjNWUwMWNiNTUwZTVhMTE2OWE0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.BqcFoSkxfJ--qeUbCc4MGycpBXOez5-Y5IOVCWgLwhI)

See [Run secured archive services on a single host - Verify OIDC client for Wildfly Administration Console in Keycloak](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Run-secured-archive-services-on-a-single-host#verify-oidc-client-for-wildfly-administration-console-in-keycloak)
> Only users with role `ADMINISTRATOR` are permitted to access the WildFly Administration Console.
